src/layouts/Sidebar.tsx

```diff
@@ -385,7 +385,10 @@ const Sidebar = () => {
     refetchInterval: 60000,
   });
 
-  const rawAnnouncements = getAnnouncementsQuery.data?.data ?? [];
+  const rawAnnouncements = useMemo(
+    () => getAnnouncementsQuery.data?.data ?? [],
+    [getAnnouncementsQuery.data?.data],
+  );
```

Comment:
I don't know if this is really needed, this will prevent a few renders when data is undefined
But we have a lot of code like this, and this seems like a micro optimization